randovania/games/prime1/exporter/game_exporter.py

```diff
@@ -197,3 +212,13 @@ def export_game(self, patch_data: dict, export_params: GameExportParams,
                 raise
             else:
                 raise RuntimeError(f"randomprime panic: {e}") from e
+
+
+        if any(v != 1.0 for v in [random_scale_min, random_scale_max,
+                                  random_health_min, random_health_max,
+                                  random_speed_min, random_speed_max,
+                                  random_damage_min, random_damage_max,
+                                  random_knockback_min, random_knockback_max]):
+            PARAMETEREDITOR.PARAMETEREDITOR.PyPARAMETEREDITOR(new_config["inputIso"], new_config["outputIso"], patch_data["seed"], random_scale_min, random_scale_max, random_health_min,
```

Comment:
Are you overwriting outputIso with a copy of inputIso? This will throw away any changes made by randomprime.

Reply:
randomprime changes are not being clobbered